Automattic/wp-calypso
# Update related posts assets

#### Changes proposed in this Pull Request

* Updates the assets for related posts in `/public` to the latest versions
* References the assets from `/public` in the component
* Removes the unused `client/assets` dir

#### Testing instructions

* Go to http://calypso.localhost:3000/settings/traffic/[siteSlug]
* Enable thumbnails for related posts
* Check that the thumbnails for the related posts examples still load.
